### ISSUE TITLE
eve energy commision failes

### ISSUE THREAD

**Issue by alexanderniebuhr** [home-assistant-libs/python-matter-server#162]
comission tried after resseting:

```
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Setting attestation nonce to random value
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Setting CSR nonce to random value
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Starting commissioning discovery over BLE
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Starting commissioning discovery over DNS-SD
2023-01-08 09:48:23 home-assistant chip.BLE[1] INFO BLE removing known devices.
2023-01-08 09:48:23 home-assistant chip.BLE[1] INFO BLE initiating scan.
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Discovered device to be commissioned over DNS-SD
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Attempting PASE connection to UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540
2023-01-08 09:48:23 home-assistant chip.EM[1] INFO <<< [E:61908i M:71926605] (U) Msg TX to 0:0000000000000000 [0000] --- Type 0000:20 (SecureChannel:PBKDFParamRequest)
2023-01-08 09:48:23 home-assistant chip.IN[1] INFO (U) Sending msg 71926605 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Commission called for node ID 0x0000000000000001
2023-01-08 09:48:23 home-assistant chip.CTL[1] INFO Discovered device to be commissioned over DNS-SD
2023-01-08 09:48:28 home-assistant chip.IN[1] INFO (U) Sending msg 71926605 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:48:32 home-assistant chip.IN[1] INFO (U) Sending msg 71926605 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:48:33 home-assistant chip.CTL[1] ERROR Commissioning discovery over BLE failed: ../src/platform/Linux/BLEManagerImpl.cpp:787: CHIP Error 0x00000032: Timeout
2023-01-08 09:48:33 home-assistant chip.-[1] ERROR ../src/platform/Linux/BLEManagerImpl.cpp:787: CHIP Error 0x00000032: Timeout at ../src/controller/SetUpCodePairer.cpp:283
2023-01-08 09:48:40 home-assistant chip.IN[1] INFO (U) Sending msg 71926605 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:48:51 home-assistant chip.IN[1] INFO (U) Sending msg 71926605 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:48:53 home-assistant chip.CTL[1] ERROR Discovery timed out
2023-01-08 09:49:09 home-assistant chip.EM[1] ERROR Failed to Send CHIP MessageCounter:71926605 on exchange 61908i sendCount: 4 max retries: 4
2023-01-08 09:49:13 home-assistant chip.SC[1] ERROR PASESession timed out while waiting for a response from the peer. Expected message type was 33
2023-01-08 09:49:13 home-assistant chip.CTL[1] INFO Ignoring SecurePairingFailed status for now; we have more discovered devices to try
2023-01-08 09:49:13 home-assistant chip.CTL[1] INFO Attempting PASE connection to UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540
2023-01-08 09:49:13 home-assistant chip.EM[1] INFO <<< [E:61909i M:71926606] (U) Msg TX to 0:0000000000000000 [0000] --- Type 0000:20 (SecureChannel:PBKDFParamRequest)
2023-01-08 09:49:13 home-assistant chip.IN[1] INFO (U) Sending msg 71926606 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:49:13 home-assistant chip.CTL[1] INFO Commission called for node ID 0x0000000000000001
2023-01-08 09:49:13 home-assistant chip.CTL[1] INFO Trying connection to commissionee over different transport
2023-01-08 09:49:18 home-assistant chip.IN[1] INFO (U) Sending msg 71926606 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:49:23 home-assistant chip.IN[1] INFO (U) Sending msg 71926606 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:49:32 home-assistant chip.IN[1] INFO (U) Sending msg 71926606 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:49:44 home-assistant chip.IN[1] INFO (U) Sending msg 71926606 to IP address 'UDP:[fda7:b5a6:3fcc:0:a1d2:d04f:9a29:4bf9]:5540'
2023-01-08 09:50:02 home-assistant chip.EM[1] ERROR Failed to Send CHIP MessageCounter:71926606 on exchange 61909i sendCount: 4 max retries: 4
2023-01-08 09:50:03 home-assistant chip.SC[1] ERROR PASESession timed out while waiting for a response from the peer. Expected message type was 33
2023-01-08 09:50:03 home-assistant matter_server.server.client_handler[1] ERROR [140079894612816] Error handling message: CommandMessage(message_id='efca6ba8ad2e4b7ebfb302bc4e700f44', command='commission_with_code', args={'code': '3336-271-0925'})
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/matter_server/server/client_handler.py", line 185, in _run_handler
    result = await result
  File "/usr/local/lib/python3.9/dist-packages/matter_server/server/device_controller.py", line 122, in commission_with_code
    raise NodeCommissionFailed(
matter_server.common.models.error.NodeCommissionFailed: Commission with code failed for node 1
```

**Comment by noname202** [comment]
I'm experiencing the same issue. Aside from that, DNS-Discovery is working fine for my other devices.

**Comment by marcelveldt** [comment]
This one will be fixed in HA patch release 2023.1.6
